src/test/javascript/spec/component/team/team-owner-search.component.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { CourseManagementService } from 'app/course/manage/course-management.service';
+import { TeamOwnerSearchComponent } from 'app/exercises/shared/team/team-owner-search/team-owner-search.component';
+import { MockCourseManagementService } from '../../helpers/mocks/service/mock-course-management.service';
+import { ArtemisTestModule } from '../../test.module';
+import { User } from 'app/core/user/user.model';
+import { NgbTypeahead } from '@ng-bootstrap/ng-bootstrap';
+import { MockDirective, MockPipe } from 'ng-mocks';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+import { NgModel } from '@angular/forms';
+import { HttpResponse } from '@angular/common/http';
+import { of, throwError } from 'rxjs';
+
+describe('Team Owner Search Component', () => {
+    let comp: TeamOwnerSearchComponent;
+    let fixture: ComponentFixture<TeamOwnerSearchComponent>;
+    let courseService: CourseManagementService;
+
+    const owner = { login: 'userLogin', name: 'name' } as User;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule],
+            declarations: [TeamOwnerSearchComponent, MockDirective(NgbTypeahead), MockDirective(NgModel), MockPipe(ArtemisTranslatePipe)],
+            providers: [{ provide: CourseManagementService, useClass: MockCourseManagementService }],
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(TeamOwnerSearchComponent);
+        comp = fixture.componentInstance;
+        courseService = fixture.debugElement.injector.get(CourseManagementService);
```

Comment:
Should we use ``TestBed.inject()`` here for brevity?

Reply:
With the current component under test that would also be fine I guess, but according to our previous discussion at https://github.com/ls1intum/Artemis/pull/4686#discussion_r808212210 I am still a bit reluctant that this brevity is worth having the low but extra risk. Maybe we can decide on a convention for injections in general.

Reply:
Well, I think, as long as it works, what's wrong about keeping it short? You would notice if it doesn't. In fact, we do use this longer, more verbose method 283 times in our test code, but ``TestBed.inject`` is used 684 times without breaking anything.
But you're right, we don't have guidelines on this, and this works, so we can keep it